ray/helm/ray/values.yaml

```diff
@@ -101,7 +101,7 @@ workers:
 - groupName: small-group
   image:
     repository: dkr.plural.sh/ray/rayproject/ray
-    tag: 2.0.0
+    tag: 2.5.0
```

Comment:
is there a reason this is not `2.7.0` too?

Reply:
Nope, this is causing my issue haha